impexp-client/src/main/java/org/citydb/modules/kml/gui/preferences/GeneralPanel.java

```diff
@@ -406,6 +433,8 @@ public void setSettings() {
 		kmlExporter.setPathOfGltfConverter(gltfConverterBrowseText.getText());
 		kmlExporter.setNotCreateColladaFiles(notCreateColladaCheckbox.isSelected());
 		kmlExporter.setEmbedTexturesInGltfFiles(embedTexturesInGltfCheckbox.isSelected());
+		kmlExporter.setExportGltfBinary(exportGltfBinary.isSelected());
+		kmlExporter.setEnableGltfDracoCompression(enableGltfDracoCompression.isSelected());
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).